emails/tests/models_tests.py

```diff
@@ -450,6 +450,12 @@ def test_subdomain_available_taken_returns_False(self):
         premium_profile.add_subdomain('thisisfine')
         assert Profile.subdomain_available('thisisfine') == False
 
+    def test_subdomain_available_with_space_returns_False(self):
+        assert Profile.subdomain_available('my domain') == False
+
+    def test_subdomain_available_with_special_char_returns_False(self):
+        assert Profile.subdomain_available('my@domain') == False
+
```

Comment:
```suggestion
    def test_subdomain_available_with_special_char_returns_False(self):
        assert Profile.subdomain_available('my@domain') == False

    def test_subdomain_available_with_dash_returns_True(self):
        assert Profile.subdomain_available('my-domain') == True

    def test_subdomain_available_with_dash_at_front_returns_False(self):
        assert Profile.subdomain_available('-mydomain') == False
```
Maybe not necessary but since we are allowing `-` I thought we should add test case for that.